src/test/resources/org/openmrs/uitestframework/test-cypress.properties

```diff
@@ -1 +1 @@
-webapp.url=https://openmrs-spa.org/openmrs/spa
+webapp.url=http://localhost:8081/openmrs/spa
```

Comment:
Does this file actually get picked up anywhere? Cypress is usually configured either via the command line or the `cypress.json` file.

Reply:
Moved it to a properties file as per the instructions given by Kaweesi. The intention was to keep all the properties in a single place. The file is been loaded in `cypress/plugins/index.js` file.  An additional cypresses task is written `cypress/support/commands.js` to visit a page.

However, having the base URL configured in the  `cypress.json` seems pretty straightforward. What do you think?

Reply:
Yes, I think configuring it in `cypress.json` is fine.